### PR TITLE
update expo to 49

### PR DESCRIPTION
**Pull request by benschac**
- https://blog.expo.dev/expo-sdk-49-c6d398cdf740 went through this
- ran `npx expo-doctor@latest`
- remove eas extra, simulator does run w/o project id.
- change project place holder, no longer valid
- fixed all issues in mono repo deps
- tested on ios and web

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1772** <a href="https://app.graphite.dev/github/pr/tamagui/tamagui/1772" target="_blank">Graphite</a> 
    * **PR #1773** <a href="https://app.graphite.dev/github/pr/tamagui/tamagui/1773" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/tamagui/tamagui/1773?utm_source=stack-comment).

File: starters/next-expo-solito/apps/expo/app.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "expo": {
-    "name": "[yourprojectsname]",
-    "slug": "[yourprojectsname]",
-    "scheme": "[yourprojectsname]",
+    "name": "yourprojectsname",
```
Comment (natew) [inline starters/next-expo-solito/apps/expo/app.json:3]: are we doing find/replace in cli or no?

Comment (benschac) [reply starters/next-expo-solito/apps/expo/app.json:3]: No. This _should_ be it. was just addressing warnings / linting warning coming from the bump to 49.

Comment (benschac) [reply starters/next-expo-solito/apps/expo/app.json:3]: did a quick search and updated a startup script. tyty

**Pull request merged**